### PR TITLE
Pi ctrl cannot set acceleration

### PR DESCRIPTION
**Pull request by PI-SRau**
if controller does not know how to set velocity, setting acceleration and deceleration must be disabled also

### CODE REVIEW

File: motorApp/PIGCS2Src/PIGCSMotorController.cpp [hunk 1]
```diff
@@ -163,6 +163,10 @@ asynStatus PIGCSMotorController::getStatus(PIasynAxis* pAxis, int& homing, int&
 
 asynStatus PIGCSMotorController::getMaxAcceleration(PIasynAxis* pAxis)
 {
+    if (!m_KnowsVELcommand)
+    {
+        return asynSuccess;
+    }
 	double maxAcc, maxDec;
```
Comment (tboegi) [inline motorApp/PIGCS2Src/PIGCSMotorController.cpp]: The indentation looks weird, at least in my browser

Comment (PI-SRau) [inline motorApp/PIGCS2Src/PIGCSMotorController.cpp]: this source file has a mix of spaces and tabs used for indentation. Most of the file uses tabs (displayed as 8 spaces), some parts use 4 spaces for one level of indentation.
 What is the epics standard?

Comment (tboegi) [inline motorApp/PIGCS2Src/PIGCSMotorController.cpp]: To my knowledge: convert all tabs to spaces to achieve a consistent view in all editors.

Comment (MarkRivers) [inline motorApp/PIGCS2Src/PIGCSMotorController.cpp]: EPICS does not really have a standard, but 2 or 4 spaces (no tabs) is what is recommended by most style guides and is what I use in all of my code.

**Comment by rsluiter** [comment]
Steffen, your instructions to Chris were to,
"...test this by changing the code. Add
    if (!m_KnowsVELcommand)
    {
        return asynSuccess;
    }
to the beginning of PIGCSMotorController::getMaxAcceleration and PIGCSMotorController::setAcceleration"

Your pull request has the change to getMaxAcceleration, but not setAcceleration.
Is the change to setAcceleration() not needed?

**Comment by PI-SRau** [comment]
I changed all tabs to spaces

**Comment by PI-SRau** [comment]
Ther are two calls changin the acceleration: setAccelerationCts and setAcceleration. One takes the acceleration in counts and calls setAcceleration taking the valus in eng. units. ONly the second one actually communicates and has the check.

**Pull request merged**